src/Sulu/Bundle/ContentBundle/phpunit.xml.dist

```diff
@@ -18,8 +18,7 @@
     </filter>
 
     <php>
-        <server name="KERNEL_DIR" value="Tests/app"/>
-        <var name="APP_DB" value="mysql"/>
```

Comment:
Is that the default value? Or why can we remove that everywhere?

Reply:
@danrot `APP_DB` and could not find anything about this var